docs_src/microservices/device/Ch-DeviceServices.md

```diff
@@ -2,40 +2,144 @@
+
+### Device Service Functional Requirements
+
+[Requirements for the device service](https://wiki.edgexfoundry.org/download/attachments/329488/edgex-device-service-requirements-v11.pdf) are provided in the Edge Wiki. These
 requirements are being used to define what functionality needs to be
 offered via any Device Service SDK to produce the device service
-scaffolding code. They may also help the reader understand the duties
+scaffolding code. They may also help the reader further understand the duties
 and role of a device service.
 
-[DS-SDK
-Functional Requirements](https://wiki.edgexfoundry.org/download/attachments/329488/edgex-device-service-requirements-v11.pdf)
+## Device Profile
+
+EdgeX comes with a number of existing device services for communicating with devices that speak many IoT protocols – such as Modbus, BACnet, BLE, etc.  While these devices services know how to speak to devices that communicate by the associated protocol, the device service doesn’t know the specifics of all devices that speak that protocol.  For example, there are thousands of Modbus devices in the world.  It is a common industrial protocol used in a variety of devices.  Some Modbus devices measure temperature and humidity and provide thermostatic control over building HVAC systems, while other Modbus devices are used in automation control of flare gas meters in the oil and gas industry.  This diversity of devices means that the Modbus device service could never know how to communicate with each Modbus device directly.  The device service just knows the Modbus protocol generically and must be informed of how to communicate with each individual device based on what that device knows and communicates.  Using an analogy, you may speak a language or two.  Just because you speak English, doesn’t mean you know everything about all English-speaking people.  For example, just because someone spoke English, you would not know if they could solve a calculus problem for you or if they can sing your favorite song.
+
+Device profiles describe a specific device to a device service.  Each device managed by a device service has an association device profile, which defines that device in terms of the data it reports and operations that it supports. General characteristics about the type of device, the data the device provides, and how to command the device is all provided in a device profile. A device profile is described in YAML which is a human-readable data serialization language (similar to a markup language like XML).  See the [page on device profiles](./profile/Ch-DeviceProfile.md) to learn more about how they provide the detail EdgeX device services need to communicate with a device.
```

Comment:
A device profile is **normally** described in YAML.
It can also be described in JSON.